skimage/morphology/convex_hull.py

```diff
@@ -93,40 +93,66 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
     return mask
 
 
-def convex_hull_object(image, neighbors=8):
-    """Compute the convex hull image of individual objects in a binary image.
+def convex_hull_object(image, neighbors=None, connectivity=None):
```

Comment:
Hmm, I am not sure what asterisk is doing here.

Reply:
@jni has been wanting to have keyword only arguments for many functions.

This is a tip to him.

Reply:
Oh, well. The asterisk has to be in a different place then.
I suggest to have a SKIP for that first. 🙂

Reply:
no, putting the asterix before neighbors would make people using positional arguments error instead of creating a warning. 

Honestly, if the library wide deprecation were to happen, then 
https://github.com/hmaarrfk/wabisabi/blob/master/wabisabi/kwonly_change.py
should be used.

I think the change is REALLY intrusive though.

This way, it creates a very non-intruisive change.

Reply:
but helps users be more explicit.

Reply:
No need to have a SKIP for this. This is something we started to do on all new functions/parameters.

Reply:
Alright, let's change! I was not aware of https://www.python.org/dev/peps/pep-3102/ 😕.